src/main/webapp/dev/js/main/instructorStudentList.es6

```diff
@@ -96,9 +96,10 @@ function filterSection() {
  */
 function filterTeam() {
     $('input[id^="team_check"]').each(function () {
-        const courseIdx = $(this).attr('id').split('-')[1];
-        const sectionIdx = $(this).attr('id').split('-')[2];
-        const teamIdx = $(this).attr('id').split('-')[3];
+        const $thisAttr = $(this).attr('id').split('-');
```

Comment:
This is a generic array and no longer a jQuery object, as such do not prefix it with an `$`.
Also, the variable name could be improved. `thisAttr` is too generic; what kind of `attr` (it is very specific here)? In addition, the variable name should suggest that it is an array.

Reply:
On second though teamAttributes is wrong. Would `indexArray` be better?